journal/202108.md

```diff
@@ -0,0 +1,424 @@
+# {DRAFT} Decred Journal – August 2021
+
+![{alt text}](../img/{file.ext} "{hover tooltip text}")
+
+_Image: {"$title by $author. $optional-description" for artwork, "$description" for photos}_
+
+{ intro, major news of the month, don't put any links here
+
+- Politeia v1.1.0 release
+- again solid dev progress in multiple projects
+
+}
+
+Contents: { sync }
+
+- [Development](#development)
+- [People](#people)
+- [Governance](#governance)
+- [Network](#network)
+- [Ecosystem](#ecosystem)
+- [Outreach](#outreach)
+- [Events](#events)
+- [Media](#media)
+- [Discussions](#discussions)
+- [Markets](#markets)
+- [Relevant External](#relevant-external)
+
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can [build and run](https://medium.com/@artikozel/the-decred-node-back-to-the-source-part-one-27d4576e7e1c), but is not yet available in release binaries for regular users.
+
+<a id="dcrd" />
+
+**[dcrd](https://github.com/decred/dcrd)**
+
+Implementation of the [Revert Treasury Expenditure Policy](https://github.com/decred/dcps/blob/master/dcp-0007/dcp-0007.mediawiki) consensus change (DCP-7) has been [merged](https://github.com/decred/dcrd/pull/2680) (following some [preparations](https://github.com/decred/dcrd/pull/2679) to make the it easier to review). If the vote passes, spend limit will be based on treasury's historical income and not its historical spends, which in turn unlocks the ability to use the new decentralized treasury.
+
+Merged work towards the [Automatic Ticket Revocations](https://github.com/decred/dcps/blob/master/dcp-0009/dcp-0009.mediawiki) consensus change (DCP-9):
+
+- added a method to get all tickets that [will expire](https://github.com/decred/dcrd/pull/2701) in the next block
+- unified vote and revocation [validation](https://github.com/decred/dcrd/pull/2702)
+- added a function to create [revocation](https://github.com/decred/dcrd/pull/2707) tx for a given ticket
+
+Other merged work:
+
+- optimized [precomputed](https://github.com/decred/dcrd/pull/2690) math values for ~10% faster signature verification and ~400 KiB smaller `dcrd` binary
+- rewritten [NAF conversion](https://github.com/decred/dcrd/pull/2695) to avoid all heap allocations and switch to a faster algorithm, removing another ~1% from signature verification time
+  - { maybe mention memory savings too? if yes, how to present "alloc(s)/op" in the benchmark? }
```

Comment:
This is the relevant line from the benchmark:
```
NAF          2.00 ± 0%       0.00            -100.00%  (p=0.008 n=5+5)
```

However, I think you already called that out by saying "to avoid all heap allocations".